src/landing/home-stats/home-stats-table.jsx

```diff
@@ -163,42 +164,48 @@ function getGridRowsAndColumns(stats) {
     }
   }
 
-  if (stats.topLists.blocked?.length) {
-    for (const blocked of stats.topLists.blocked) {
+  if (stats.topLists.blocked && Object.keys(stats.topLists.blocked).length) {
+    for (const [key, value] of Object.entries(stats.topLists.blocked)) {
       blockedData['topBlocked'].push({
-        ...blocked,
-        block_count: blocked.block_count.toLocaleString(),
+        did: key,
+        count: value.count.toLocaleString()
       });
     }
+
+    blockedData['topBlocked'].sort((a, b) => parseInt(b.count) - parseInt(a.count))
   }
 
-  if (stats.topLists.blocked24?.length) {
-    for (const blocked of stats.topLists.blocked24) {
+  if (stats.topLists.blocked24 && Object.keys(stats.topLists.blocked24).length) {
+    for (const [key, value] of Object.entries(stats.topLists.blocked24)) {
       blockedData['topBlocked24'].push({
-        ...blocked,
-        block_count: blocked.block_count.toLocaleString(),
+        did: key,
+        count: value.count.toLocaleString()
       });
     }
+
+    blockedData['topBlocked24'].sort((a, b) => parseInt(b.count) - parseInt(a.count))
   }
 
-  if (stats.topLists.blockers?.length) {
-    for (const blocked of stats.topLists.blockers) {
+  if (stats.topLists.blockers && Object.keys(stats.topLists.blockers).length) {
+    for (const [key, value] of Object.entries(stats.topLists.blockers)) {
       blockedData['topBlockers'].push({
-        Handle: blocked.Handle,
-        block_count: blocked.block_count.toLocaleString(),
-        ProfileURL: blocked.ProfileURL,
-        did: blocked.did
+        did: key,
+        count: value.count.toLocaleString()
       });
     }
+
+    blockedData['topBlockers'].sort((a, b) => parseInt(b.count) - parseInt(a.count))
   }
 
-  if (stats.topLists.blockers24?.length) {
-    for (const blocked of stats.topLists.blockers24) {
+  if (stats.topLists.blockers24 && Object.keys(stats.topLists.blockers24).length) {
+    for (const [key, value] of Object.entries(stats.topLists.blockers24)) {
       blockedData['topBlockers24'].push({
-        ...blocked,
-        block_count: blocked.block_count.toLocaleString(),
+        did: key,
+        count: value.count.toLocaleString()
       });
     }
+
+    blockedData['topBlockers24'].sort((a, b) => parseInt(b.count) - parseInt(a.count))
```

Comment:
```suggestion
    blockedData['topBlockers24'].sort((a, b) => b.count - a.count)
```
no need to parse now that the returns are numbers